virtio-devices/src/transport/pci_device.rs

```diff
@@ -287,13 +287,14 @@ struct VirtioPciDeviceState {
 
 impl VersionMapped for VirtioPciDeviceState {}
 
+type QueueTuples = Vec<(usize, Queue<GuestMemoryAtomic<GuestMemoryMmap>>, EventFd)>;
```

Comment:
May want to move it to `virtio-devices/src/lib.rs` to be reused across the crate? Also may worth using a struct for a bit better readability?

Reply:
I'm gonna override the clippy warning.